etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/messages/SendMessageUseCase.java

```diff
@@ -0,0 +1,13 @@
+package gov.hhs.cdc.trustedintermediary.etor.messages;
+
+/**
+ * This interface represents a use case for sending a generic message. It provides a method to
+ * convert and send the message.
+ *
+ * @param <T> the type of message to be sent
```

Comment:
Is this a param?

Reply:
Generics use the same @param tag in Javadocs: https://stackoverflow.com/a/2015998